.github/workflows/pull-request.yml

```
@@ -243,63 +241,106 @@ jobs:
       - name: Get changed TypeScript files
         id: changed-files
         uses: tj-actions/changed-files@v45
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)